### DISCUSSION TITLE
can't redefine non-configurable property "$persist"

### DISCUSSION THREAD

**Discussion by swarakaka** [livewire/livewire#6608] [1. Help]
After installing and configuring livewire v3, the following error appears in the console:

`Uncaught TypeError: can't redefine non-configurable property "$persist"
    src_default http://hotel-monitoring.localhost:5174/vendor/livewire/livewire/dist/livewire.esm.js:4712
    plugin http://hotel-monitoring.localhost:5174/vendor/livewire/livewire/dist/livewire.esm.js:2733
    plugin http://hotel-monitoring.localhost:5174/vendor/livewire/livewire/dist/livewire.esm.js:2733
    start http://hotel-monitoring.localhost:5174/vendor/livewire/livewire/dist/livewire.esm.js:7458
    <anonymous> http://hotel-monitoring.localhost:5174/resources/js/app.js:133
[livewire.esm.js:4712:14](http://hotel-monitoring.localhost:5174/vendor/livewire/livewire/dist/livewire.esm.js)
    ObjectOrReflectDefineProperty self-hosted:2176
    defineProperty self-hosted:2241
    src_default http://hotel-monitoring.localhost:5174/vendor/livewire/livewire/dist/livewire.esm.js:4712
    plugin http://hotel-monitoring.localhost:5174/vendor/livewire/livewire/dist/livewire.esm.js:2733
    forEach self-hosted:203
    plugin http://hotel-monitoring.localhost:5174/vendor/livewire/livewire/dist/livewire.esm.js:2733
    start http://hotel-monitoring.localhost:5174/vendor/livewire/livewire/dist/livewire.esm.js:7458
    <anonymous> http://hotel-monitoring.localhost:5174/resources/js/app.js:133`

**Comment by JSFTI** [comment]
I think this error is thrown if the project accidentally imported the Livewire JavaScript code twice. I encountered this error because my project initializes Livewire from a custom TS file, and Livewire automatically injects a Livewire Script again in the body. [To fix this, I changed @livewireScript to @livewireScriptConfig](https://livewire.laravel.com/docs/installation#manually-bundling-livewire-and-alpine).

For some reason, changing 'inject_assets' to false in config/livewire.php does not work. Livewire still injects the script tag. I don't know if this is a bug, a misunderstanding on my part, or a bad cache in my project.
